src/test/java/com/Saurabh/GitHubAiAPI/StringOperationsTest.java

```diff
@@ -0,0 +1,101 @@
+package com.Saurabh.GitHubAiAPI;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class StringOperationsTest {
+
+
+    @BeforeAll
+    public static void setUp() {
+        // TO DO
+    }
```

Comment:
The comment 'test to check if the count of vwels is correct' contains a spelling error; 'vwels' should be 'vowels'.